### PR TITLE
Updated IBM docs

### PR DESCRIPTION
**Pull request by Nadine2016**

### CODE REVIEW

File: content/portworx-install-with-kubernetes/cloud/ibm/_index.md [hunk 1]
```diff
@@ -17,44 +17,42 @@ Before you begin:
...
 
 **How can I make sure that my data is stored highly available?** </br>
 You need at least 3 worker nodes in your Portworx cluster so that Portworx can replicate your data across nodes. By replicating your data across worker nodes, Portworx can ensure that your stateful app can be rescheduled to a different worker node in case of a failure without losing data. For even higher availability, use a [multizone cluster](https://cloud.ibm.com/docs/containers?topic=containers-plan_clusters#multizone) and replicate your volumes on SDS worker nodes across 3 zones.
 
-## Step 2: Creating or preparing your cluster for Portworx
+**Can I install Portworx on all supported infrastructure providers and container platforms?** </br>
+Portworx is supported only in classic IBM Cloud Kubernetes Service clusters, and is not available in VPC on Classic or Red Hat® OpenShift® on IBM Cloud™ clusters.
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/cloud/ibm/_index.md]: Portworx is supported on `Red Hat® OpenShift® on IBM Cloud™`

File: content/portworx-install-with-kubernetes/cloud/ibm/_index.md [hunk 2]
```diff
@@ -17,44 +17,42 @@ Before you begin:
...
+## Step 2: Create an IBM Cloud Kubernetes Service cluster
 
-2. If you want to create a [multizone cluster](https://cloud.ibm.com/docs/containers?topic=containers-plan_clusters#multizone) for high availability, enable [VLAN spanning](https://cloud.ibm.com/docs/infrastructure/vlans?topic=vlans-vlan-spanning#vlan-spanning) for your IBM Cloud account.
-
-3. [Create or use an existing cluster](https://cloud.ibm.com/docs/containers?topic=containers-clusters#clusters_ui) in IBM Cloud Kubernetes Service with a Kubernetes version of 1.10 or higher. To ensure high availability for your data, set up a [multizone cluster](https://cloud.ibm.com/docs/containers?topic=containers-plan_clusters#multizone) with at least 3 worker nodes and spread the worker nodes across zones.
+To install Portworx, you must have an IBM Cloud Kubernetes Service cluster that runs Kubernetes version 1.10 or higher. To make sure that your cluster is set up with worker nodes that offer best performance for you Portworx cluster, review Step 1: Choosing the right worker node flavor for your IBM Cloud Kubernetes Service cluster for Portworx.
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/cloud/ibm/_index.md]: We can remove the version requirement. Installation will work on all the current versions of IKS and openshift on Classic infrastructure.

File: content/portworx-install-with-kubernetes/cloud/ibm/_index.md [hunk 3]
```diff
@@ -17,44 +17,42 @@ Before you begin:
...
-
-Follow the [IBM Cloud Kubernetes Service documentation](https://cloud.ibm.com/docs/containers?topic=containers-portworx#portworx_database) to set up your IBM Compose or Databases for etcd key-value store for Portworx.
+In order for your Portworx cluster to be highly available, you must ensure that the Portworx key-value store is set up highly available. By using an instance of [Databases for etcd](https://cloud.ibm.com/docs/containers?topic=containers-portworx#portworx_database), you can set up a highly available key-value store for your Portworx cluster. Each service instance contains three etcd data members that are added to a cluster. The etcd data members are spread across zones in an IBM Cloud location and data is replicated across all etcd data members.
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/cloud/ibm/_index.md]: `the Portworx key-value store is set up to be highly available`

File: content/portworx-install-with-kubernetes/cloud/ibm/_index.md [hunk 4]
```diff
@@ -64,9 +62,9 @@ Review the following information in the IBM Cloud Kubernetes Service documentati
...
+## Step 5: Install Portworx in your cluster
 
-Install Portworx version 1.7 with a Helm chart. The Helm chart deploys a trial version of the Portworx enterprise edition `px-enterprise` that you can use for 30 days. After the trial version expires, you must [purchase a Portworx license](/reference/knowledge-base/px-licensing/) to continue to use your Portworx cluster. In addition, [Stork](https://docs.portworx.com/portworx-install-with-kubernetes/) is installed on your Kubernetes cluster. Stork is the Portworx storage scheduler and allows you to co-locate pods with their data, and create and restore snapshots of Portworx volumes.
+Install Portworx with a Helm chart. The Helm chart deploys a trial version of the Portworx enterprise edition `px-enterprise` that you can use for 30 days. After the trial version expires, you must [purchase a Portworx license](/reference/knowledge-base/px-licensing/) to continue to use your Portworx cluster. In addition, Stork is installed on your Kubernetes cluster. Stork is the Portworx storage scheduler and allows you to co-locate pods with their data, and create and restore snapshots of Portworx volumes.
```
Comment (disrani-px) [inline content/portworx-install-with-kubernetes/cloud/ibm/_index.md]: Do we want to update this to use the catalog instead?

**Comment by nathan-px** [comment]
I moved this to a new PR to change its base to master instead of 2.1

**Pull request closed without merging**